vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -462,6 +464,13 @@ export default class SbcHeader extends Mixins(NavigationMixin) {
     this.redirectToPath(this.inAuth, `${Pages.ACCOUNT}/${this.currentAccount.id}/${Pages.SETTINGS}/transactions`)
   }
 
+  private checkAccountStatus () {
+    // redirect if accoutn status is suspended
+    if (this.currentAccount?.accountStatus && this.currentAccount?.accountStatus === 'NSF_SUSPENDED') {
```

Comment:
May be get this status from an array? I think there will be other suspended status as well like SUSPENDED.

Reply:
will update in the next PR. Now we have only one value, so its good to make a direct check